### PR TITLE
docs: add more content to cloud integration

### PR DESCRIPTION
**Pull request by sunng87**
## What's Changed in this PR

Adds more content to cloud integration docs:

- Prometheus: remove rules section, add api information
- Vector: logs
- Kafka: new content
- Alloy: new content
- MindsDB: use our extention
- InfludDB: add telegraf
- OpenTelemetry: minor update

## Checklist

- [x] Please confirm that all corresponding versions of the documents have been revised.
- [x] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/greptimecloud/integrations/kafka.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Kafka
+
+如果您正在使用 Kafka 或兼容 Kafka 的消息队列来传输可观测性数据，可以直接将数据摄取到 GreptimeDB 中。
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/greptimecloud/integrations/kafka.md:3]:
```suggestion
如果您正在使用 Kafka 或兼容 Kafka 的消息队列来传输可观测性数据，可以直接将数据写入到 GreptimeDB 中。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/greptimecloud/integrations/alloy.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Alloy
+
+Grafana Alloy 是可观测数据采集器，同时兼容 OpenTelemetry 。 GreptimeDB 可被配置
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/greptimecloud/integrations/alloy.md:3]:
```suggestion
[Grafana Alloy](https://grafana.com/docs/alloy/latest/) 是可观测数据采集器，同时兼容 OpenTelemetry 。 GreptimeDB 可被配置
```

**Review by nicecui — COMMENTED** [review]
please do not forget mindsdb.md and otlp.md in the zh docs

File: docs/greptimecloud/integrations/alloy.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+        basic_auth {
+            username = "<username>"
+            password = "<password>"
+        }
+    }
+}
+```
+
+## OpenTelemetry
```
[the remark this answers is not in the document]
Comment (sunng87) [reply docs/greptimecloud/integrations/alloy.md:26]: Good idea. I can copy this. Alloy can be used as OpenTelemetry collector

**Comment by sunng87** [comment]
Updated @nicecui PTAL

File: i18n/zh/docusaurus-plugin-content-docs/current/greptimecloud/integrations/otlp.md [hunk 4]
```diff
@@ -17,6 +17,26 @@ The request uses binary protobuf to encode the payload, so you need to use packa
 The package names may change according to OpenTelemetry, so we recommend that you refer to the official OpenTelemetry documentation for the most up-to-date information.
 :::
 
-For more information about the OpenTelemetry SDK, please refer to the official documentation for your preferred programming language.
-
-
+## OpenTelemetry Collector
+
+如果你使用单独的 Otel 收集器，我们推荐更加成熟的 [Grafana
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/greptimecloud/integrations/otlp.md:22]:
```suggestion
如果你使用单独的 OTEL 收集器，我们推荐更加成熟的 [Grafana
```

File: docs/greptimecloud/integrations/otlp.md [hunk 5]
```diff
@@ -17,5 +21,26 @@ The request uses binary protobuf to encode the payload, so you need to use packa
 The package names may change according to OpenTelemetry, so we recommend that you refer to the official OpenTelemetry documentation for the most up-to-date information.
 :::
 
-For more information about the OpenTelemetry SDK, please refer to the official documentation for your preferred programming language.
-
+## OpenTelemetry Collector
+
+We recommend [Grafana Alloy](https://grafana.com/docs/alloy/latest/) as Otel
```
Comment (nicecui) [inline docs/greptimecloud/integrations/otlp.md:26]:
```suggestion
We recommend [Grafana Alloy](https://grafana.com/docs/alloy/latest/) as OTEL
```

File: docs/greptimecloud/integrations/otlp.md [hunk 6]
```diff
@@ -17,5 +21,26 @@ The request uses binary protobuf to encode the payload, so you need to use packa
 The package names may change according to OpenTelemetry, so we recommend that you refer to the official OpenTelemetry documentation for the most up-to-date information.
 :::
 
-For more information about the OpenTelemetry SDK, please refer to the official documentation for your preferred programming language.
-
+## OpenTelemetry Collector
```
Comment (nicecui) [inline docs/greptimecloud/integrations/otlp.md:24]: Duplicate content leads to additional maintenance cost. We may support a component in the future to solve this problem. @sunchanglong

**Review by nicecui — APPROVED** [review]

**Pull request merged**